src/track/track.cpp

```diff
@@ -251,6 +251,10 @@ void Track::setReplayGain(const mixxx::ReplayGain& replayGain) {
     }
 }
 
+void Track::analysisFinished() {
+    emit analyzed();
+}
```

Comment:
I'm not a fan of these reorderings. The obscure the git history for now reason.

Reply:
They make the code a lot easier to understand by grouping related code. Reducing the barrier of entry to new developers is "no reason"?